modules/update-custom-certificate-central.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: PROCEDURE
+[id="update-custom-certificate-central_{context}"]
+= Updating the custom certificate for an existing instance
+
+[role="_abstract"]
+If you are using a custom certificate for Central, you can update it by following the instructions in this section.
```

Comment:
I've forgotten the term for unnecessary use of "to be", but I think you could go with "If you use a custom..."

For the second clause in the sentence, I think you can avoid self-referential text.  From ISG:

> Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". 

Maybe "..., you can update the certificate by performing the following procedure."

Rando free comment: Doesn't everyone know that you can skip the role assignment and the first para is automatically set as the abstract?